functions/inputs/from.go

```diff
@@ -99,6 +104,68 @@ type FromProcedureSpec struct {
 	AggregateMethod string
 }
 
+func newFromProcedure(qs flux.OperationSpec, pa plan.Administration) (plan.ProcedureSpec, error) {
```

Comment:
This is not new code, I just moved it so all `FromProcedureSpec` methods are together.